dev-utils/run-script.js

```diff
@@ -57,15 +46,9 @@ function runScript(scripts, scriptName, scriptArgs) {
   }
 }
 
-var scripts = {
+const scripts = {
   runUnitTests,
-  generateNotice,
-  launchSauceConnect: function launchSauceConnect() {
```

Comment:
`launchSauceConnect ` is useful for debugging saucelabs issues locally, see my comment above!